README.md

```diff
@@ -1,2 +1,5 @@
+[![pylint_test](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/pylint.yml/badge.svg)](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/pylint.yml)
+[![coveralls_test](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/coveralls_test.yml/badge.svg?branch=developer)](https://github.com/EgorHubBrest/Human_Resourse_Managment/actions/workflows/coveralls_test.yml)
```

Comment:
Please replace the "pylint_test" and "coveralls_test" workflows with the "build" workflow

Reply:
Ok,I will change